conn.go

```diff
@@ -781,10 +781,18 @@ func (w *writeCoalescer) stop() {
 	w.cond.L.Lock()
 	defer w.cond.L.Unlock()
 
+	if w.writeCh == nil {
+		return
+	}
+
 	w.flushLocked()
 	// nil the channel out sends block forever on it
 	// instead of closing which causes a send on closed channel
 	// panic.
+
+	w.quit <- struct{}{}
```

Comment:
I have this pull request patched into my systems and am still sometimes getting "freezes". When I `SIGABRT` one o the goroutines that I can't explain (of the many that are left hanging around) is blocked on this line:

```
goroutine 55 [chan send, 66 minutes]:                                           
github.com/gocql/gocql.(*writeCoalescer).stop(0xc000100300)                     
    external/com_github_gocql_gocql/conn.go:793 +0x9d                           
github.com/gocql/gocql.(*writeCoalescer).writeFlusher(0xc000100300, 0xc9c300, 0xc0000a0000, 0x30d40)
    external/com_github_gocql_gocql/conn.go:840 +0x395                          
created by github.com/gocql/gocql.newWriteCoalescer                             
    external/com_github_gocql_gocql/conn.go:734 +0x164                          
```

Not clear if that's a bad thing or not, but seemed worth noting.